mlflow/store/db_migrations/versions/bd07f7e963c5_create_index_on_run_uuid_for_postgres.py

```diff
@@ -0,0 +1,30 @@
+"""create index on run_uuid for postgres
+
+Revision ID: bd07f7e963c5
+Revises: c48cb773bb87
+Create Date: 2022-03-03 10:14:34.037978
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "bd07f7e963c5"
+down_revision = "c48cb773bb87"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # This part of the migration is only relevant for PostgreSQL.
+    # As a fix for https://github.com/mlflow/mlflow/issues/3785, create indexes on run_uuid columns
+    # to speed up SQL operations.
+    bind = op.get_bind()
+    if bind.engine.name == "postgresql":
```

Comment:
We might need to add sqlite. https://www.sqlite.org/foreignkeys.html#fk_indexes says:

> an index should be created on the child key columns of each foreign key constraint.

Reply:
Could we add this for all DBs, including mysql? Is this specific to Postgres?